### DISCUSSION TITLE
Can You Make for 3300mAh?

### DISCUSSION THREAD

**Discussion by M-i7** [PycmShoma/IncreasedBatteryCapacity#5] [Q&A]
![Screenshot_2023-05-24-22-32-24-817_ru andr7e deviceinfohw](https://github.com/PycmShoma/IncreasedBatteryCapacity/assets/16432462/aa53fe4b-49c9-4e21-90b1-986d11b22eeb)
On latest custom Rom DoVoid ported detected as Mi9 Transparent Edition, the default is 4500mAh (not 3300mAh)

I was trying recode² and just success on kernel profile to get 3300mAh
*Power profile must decompile overlay apk, and that's to hard for me

**Comment by PycmShoma** [comment]
Attach in the next message the module you are using. I will make changes in the overlay
